docs/monitoring/monitoring.md

```diff
@@ -17,225 +18,195 @@ The following topics are covered in this section:
-
-* **Memory Size**</br>
-   Displays the heap memory used by data table to store its data/records.
-
-* **Total Size**</br>
-   Displays the collective physical memory and disk overflow space used by the data table to store its data/records.
-<!--
- **BUCKETS**</br>
-   Displays the total number of buckets in the data table.
--->
+|**CPU Usage**|Graphically presents the trend of CPU utilization by all the nodes in the cluster for the last 15	 minutes. The utilization is represented in percentage value.|
+|**Heap Usage**|Graphically presents the collective utilization of Heap Memory by all the nodes in the cluster. This  graph displays three trend lines which corresponds to the utilization of Heap Memory for the following:</br> * Storage </br> * Execution</br> * JVM </br>  |
+|**Off-Heap Usage**|Graphically presents the collective utilization of Off-Heap Memory by all the nodes in the cluster. This  graph displays two trend lines which corresponds to the utilization of Off-Heap Memory for the following: </br> * Storage </br> * Execution|
+|**Disk Space**|Graphically presents the collective utilization of disk space memory by all the nodes in the cluster.|
 
-![Dashboard](../Images/monitoring_external_table.png)
-<a id="external-table"></a>
-### External Table Statistics
-
-* **Tables Count**</br>
-	Displays the total number of external tables present in the cluster. The tooltip displays the total number of external tables.
-
-* **Name**</br>
-	Displays the name of the external table.
-
-* **Provider**</br>
-	Displays the datastores provide used when creating the external table. For example, Parquet, CSV, JDBC etc.
+<a id="member"></a>
+### Members
+In the **Members** section,  you can view, in a tabular format, the details of each locator, server, and lead member within a cluster. The details are automatically refreshed after every 5 seconds.
+![Members](../Images/Dashboard-MembersList.png)
 
-* **Source**</br>
-	For Parquet and CSV format, the path of the data file used to create the external table is displayed. For JDBC, the name of the client driver is displayed.
+This table provides member details in the following columns:
 
-<a id="member-details"></a>
-## Member Details View
 
-The Member Details View offers the following capabilities and benefits:
+| Column | Description |
+|--------|--------
+|  **Status**     |  Displays the status of the members, which can be either [Running or Stopped](#statusofmembers).    |
+|   **Member**       |   Displays a brief description of the member. Click the link in the column to view detailed description of the members along with the [Member Logs](#memberlogs). In this description, you can find information such as the IP address of the host, the current working directory, and the Process ID number.  |
+|     **Type**   |     Displays the type of the member. The type can be lead, locator, or data server. The name of the active lead member is displayed in bold letters. |
```

Comment:
shall we convert values to capital letters here ?  Like LEAD, LOCATOR or DATA SERVER